maxim/logger/portkey/client.py

```
@@ -0,0 +1,37 @@
+try:
+    from portkey_ai.api_resources.client import Portkey, AsyncPortkey
+except ImportError as e:
+    raise ImportError(
+        "The portkey-ai package is required. Please install it using pip: `pip install portkey-ai` or `uv add portkey-ai`"
+    ) from e
+
+from ..openai import MaximOpenAIAsyncClient, MaximOpenAIClient
+from ..logger import Logger
+
+
+def instrument_portkey(client: Any, logger: Logger) -> Any:
```

Ruff (0.11.9)

Dynamically typed expressions (typing.Any) are disallowed in `client`

(ANN401)